src/python/pants/backend/python/util_rules/pex.py

```diff
@@ -943,6 +944,7 @@ async def create_venv_pex(
         pex=pex.script,
         python=python.script,
         bin=FrozenDict((bin_name, venv_script.script) for bin_name, venv_script in scripts.items()),
+        venv_rel_dir=venv_rel_dir.as_posix(),
```

Comment:
Folks have a habit of doing this, but `str(venv_rel_dir)` will be Windows-proof and is actually correct today too.